site/content/docs/0.0/utilities/api.md

```diff
@@ -311,22 +319,30 @@ Output:
   .opacity-lg-100 { opacity: 1 !important; }
 }
 
-@media (min-width: 1280px) {
+@media (min-width: 1320px) {
   .opacity-xl-0 { opacity: 0 !important; }
   .opacity-xl-25 { opacity: .25 !important; }
   .opacity-xl-50 { opacity: .5 !important; }
   .opacity-xl-75 { opacity: .75 !important; }
   .opacity-xl-100 { opacity: 1 !important; }
 }
 
-@media (min-width: 1440px) {
+@media (min-width: 1640px) {
   .opacity-xxl-0 { opacity: 0 !important; }
   .opacity-xxl-25 { opacity: .25 !important; }
   .opacity-xxl-50 { opacity: .5 !important; }
   .opacity-xxl-75 { opacity: .75 !important; }
   .opacity-xxl-100 { opacity: 1 !important; }
 }
-```-->
+
+@media (min-width: 1880px) {
```

Comment:
Should be .opacity-3xl

Reply:
Fixed with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2744/commits/3e3eb6e9d0201a0b45747921e4bac1742c86071c